testplan/testing/multitest/result.py

```diff
@@ -1404,103 +1404,92 @@ def log(self, message, description=None):
         :type message: ``str`` or instance
         :param description: Text description for the assertion.
         :type description: ``str``
+        :param flag: Custom flag for the assertion, set to 'email' to
+            include message in email exporter.
+        :param flag: ``str``
         :return: ``True``
         :rtype: ``bool``
         """
-        entry = base.Log(message=message, description=description)
+        entry = base.Log(message=message, description=description, flag=flag)
         _bind_entry(entry, self)
         return entry
 
-    def passing(self, description, category=None):
-        """
-        Explicit passing assertion, can be used for noting down a single passing
-        message with no further detail. The message will be included by email
-        exporter. Most common usage is within a conditional block.
-
-        .. code-block:: python
-            if some_condition:
-                result.passing('Check passes: {}'.format(...))
-            else:
-                result.failing('Unexpected failure: {}'.format(...))
-
-        :param description: Text description of the passing assertion.
-        :type description: ``str``
-        :param category: Custom category that will be used for summarization.
-        :type category: ``str``
-        :return: ``True``
-        :rtype: ``bool``
-        """
-        entry = assertions.Pass(description, category=category)
-        _bind_entry(entry, self)
-        return entry
-
-    def failing(self, description, category=None):
+    def fail(self, description, category=None, flag=None):
```

Comment:
It's a little confusing how we now have both "category" and "flag" parameters here, but I guess the docstring explains the difference so it's not that big a deal.